### ISSUE TITLE
[REC] Balance druid doesnt use cooldowns with Shadowed Orb

### ISSUE THREAD

**Issue by noiseuu** [Hekili/hekili#1602]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

Hello!

With trinket called Shadowed Orb of Torment doesnt want use cooldown, it wants use orb yes

### How to Reproduce

1. Enter game
2. See this bug first at raid
3. Trying again at dummy

### Snapshot (Link)

https://pastebin.com/EUFEuTUJ

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/bDeJfccd6bRWpDqW7oajzm

**Comment by Hekili** [comment]
It looks like the precombat usage of the Orb is missing, but otherwise in-combat it would use it.

Your snapshot doesn't help clarify much because your Orb *is* on cooldown, so you did use it (and the addon isn't going to recommend it on cooldown).

**Comment by noiseuu** [comment]
https://i.imgur.com/rlqOHou.png

Only suggesting wrath, not trinket hmh..

**Comment by Hekili** [comment]
Screenshot doesn't help.  I need a snapshot (from the addon, ALT-SHIFT-P or ALT-SHIFT-[) where your Orb *is not* already on cooldown, because the addon does not recommend abilities that are on cooldown.

**Comment by noiseuu** [comment]
https://pastebin.com/BZp8Ywkd
i hope paste correct one

**Comment by Hekili** [comment]
You do not appear to be wearing the Cache trinket in that snapshot.

**Comment by noiseuu** [comment]
Did u answer correct one? Im talking about shadowed orb

**Comment by Hekili** [comment]
Ha, you're right, dealing with multiple tickets.

```
this entry's criteria FAILS: ( ! covenant.venthyr[true] | ( cooldown.ca_inc.ready[true] & ( cooldown.berserking.remains[0.00] < 2 | ! race.troll[true] ) ) & variable.thrill_seeker_wait[true] ) & ( ! covenant.night_fae[false] | cooldown.convoke_the_spirits.remains[0.00] < 20 ) & dot.sunfire.ticking[false] & ( dot.stellar_flare.ticking[false] | ! talent.stellar_flare.enabled[false] | spell_targets.starfire[1.00] > 3 ) & dot.moonfire.ticking[false] & ! equipped.inscrutable_quantum_device[false] | equipped.inscrutable_quantum_device[false] & cooldown.inscrutable_quantum_device.remains[0.00] > 30 & ! buff.ca_inc.up[false] | boss[false] & fight_remains[4099.72] < 40
```

The trinket, in combat, isn't recommended until/unless Sunfire is on your target, and you don't seem to be far enough into your fight to have applied it.

**Comment by noiseuu** [comment]
https://pastebin.com/gZ01ULnJ
i maked new one, do u see problem here?

**Comment by Hekili** [comment]
That one is recommending your trinket.

**Comment by noiseuu** [comment]
Yea, after that i can dps like 1 minute, and still not suggestin cd:s and trinket should be prepull
Example: https://www.raidbots.com/simbot/report/tauPoTz1toyBDa2zxYUL6C
Tormetted insight at buffs before first wrath.

**Comment by shredxt** [comment]
Looks like the precombat actions were updated to include shadowed orb of torment recently in the SimC APL: https://github.com/balance-simc/Balance-SimC/commit/f833711ddfcb06caa1cabf053b4a1dcd84361267 which has yet to be translated into Hekili

**Comment by noiseuu** [comment]
Okay, so today update?

**Comment by Hekili** [comment]
My first response noted that it's missing from the precombat APL here.  I do plan to update and add it.

**Comment by Hekili** [comment]
Today's release will add the precombat usage of the Shadowed Orb of Torment.  If there are issues afterward, please submit a new ticket with snapshots showing the circumstances of any new issues.

**Comment by noiseuu** [comment]
Same issue, now suggestin orb at prepull. But not using cooldowns when i get 90 ap

https://pastebin.com/xEZdht4F

**Comment by Hekili** [comment]
This may be addressed in 9.2.7-1.0.0b.  Let me know if you still have issues after this point.
